transport_nantes/authentication/templates/registration/password_reset_form.html

```diff
@@ -0,0 +1,45 @@
+{% extends 'asso_tn/base_transport_nantes.html' %}
+{% load static %}
+
+
+{% block app_content %}
+
+<style type="text/css">
+  .form-signin {
+    width: 100%;
+    max-width: 330px;
+    padding: 15px;
+    margin: auto;
+  }
+  .form-signin .checkbox {
+    font-weight: 400;
+  }
+  .form-signin .form-control {
+    position: relative;
+    box-sizing: border-box;
+    height: auto;
+    padding: 10px;
+    font-size: 16px;
+  }
+  .form-signin .form-control:focus {
+    z-index: 2;
+  }
+  .form-signin input[type="email"] {
+    margin-bottom: 10px;
+    border-bottom-right-radius: 0;
+    border-bottom-left-radius: 0;
+  }
+
+  .h3{
+    text-align: center;
+  }
+</style>
+
+
+<form method="POST" class="form-signin">{% csrf_token %}
+	<h1 class="h3 mb-3 font-weight-normal">Réinitialisation du mot de passe</h1>
+    <input name="email" class="form-control" placeholder="Adresse mail" type="email" id="id_email" required="true">
```

Comment:
long line

Perhaps consider using pylint?  (I've not checked if it works well for django, but there must be similar tools if it doesn't.)

Reply:
It's kind of weird for this part. I use "python-dev-tools" which a powerfull linter and formatter but it doesn't seem to work with django. I've even looked in configuration file but I can't figure ot why it's not working well.

Reply:
Well, this file is html, I wouldn't expect _this_ one to be helped by pylint...

Reply:
(So my confusing suggestion with specific problem, my bad.  But using a lint tool is still a good idea.)